gamestonk_terminal/options/op_controller.py

```diff
@@ -87,6 +89,7 @@ def print_help(self):
         print("   info          display option information (volatility, IV rank etc)")
         print("")
         print("   calc          basic call/put PnL calculator")
+        print("   act           show unusual options activity")
```

Comment:
Since this is kind of "discovery mode", can we add this right below screener option?